### PR TITLE
Add docs on how to contribute to docs

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,27 +73,27 @@ If you have questions about the project, using PyScript, or anything else, pleas
...
+Use `conda activate $environment_name` to activate your environment.
 
-```
-git checkout -b branch_name #creates new branch
-git checkout branch_name #goes to branch
-```
+To add new information to the documentation make sure you conform with PyScript's code of conduct and with the general principles of Diataxis. Don't worry about reading too much on it, just do your best to keep your contributions on the correct axis.
 
-It's not mandatory but it's advisible to have one issue that is related to your PR, the more information you give and the clearer you can communicate your ideas the faster maintainers will be able to review your changes and merge your PR.
+You need to write your documentation files using [Markedly Structured Text](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html), which is very similar to vanilla Markdown but with some addons to create the documentation infrastructure.
```
Comment (ntoll) [inline CONTRIBUTING.md:90]: Delete: `You need to write your documentation files using`
Replace with: `Write using`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -73,27 +73,27 @@ If you have questions about the project, using PyScript, or anything else, pleas
...
 
-Please try to keep the changes on your PR within the issue's scope. In case you don't have a related issue, limit the your PR's scope in a way that a short sentence summarizes its contents. This sentence should be the name of your PR. If you can't think of a sentence that summarizes your PR well, might be a sign that you need to split it in two or more PRs!
+Once your changes are done you can initialize a server and check them on the following address: http:///127.0.0.1:8000, using the command:
```
Comment (ntoll) [inline CONTRIBUTING.md:92]:
```
Once done, initial a server to check your work:

make livehtml

(Visible here: http:///127.0.0.1:8000)

**Review by ntoll — CHANGES_REQUESTED** [review]
Can we fix the failing tests..? Otherwise LGTM 🚀 👍 :shipit:

**Comment by marimeireles** [comment]
Grateful for the review @ntoll! 🌸

**Review by ntoll — APPROVED** [review]

**Pull request merged**